api/http/Server.go

```diff
@@ -30,6 +31,13 @@ func (s *Server) Serve(wait *sync.WaitGroup) error {
 		return err
 	}
 
+	handler := cors.New(cors.Options{
+		AllowedMethods:   []string{"GET", "POST", "PATCH", "PUT", "DELETE"},
```

Comment:
given that we're working with a js frontend, I guess we'd also need the `OPTIONS` Method covered. Quote from thread: `... It’s the server that needs to respond to the OPTION with the Allow-origin set with something that matches the caller’s host`

Reply:
Let's just try it out without, and add it if needed